tests/api2/test_440_snmp.py

```diff
@@ -97,6 +100,66 @@ def add_SNMPv3_user():
     yield
 
 
+@pytest.fixture(scope='function')
+def create_nested_structure():
+    """
+    Create the following structure:
+        tank -+-> dataset_1 -+-> dataset2 -+-> dataset_3
+              |-> zvol_1a    |-> zvol_2a   |-> zvol_3a
+              |-> zvol_1b    |-> zvol_2b   |-> zvol_3b
+              |-> file_1     |-> file_2    |-> file_3
+              |-> dir_1      |-> dir_2     |-> dir_3
+    TODO: Make this generic and move to assets
+    """
+    ds_path = ""
+    ds_list = []
+    zv_list = []
+    dir_list = []
+    file_list = []
+    with ExitStack() as es:
+
+        for i in range(1, 4):
+            preamble = f"{ds_path + '/' if i > 1 else ''}"
+            vol_path = f"{preamble}zvol_{i}"
+
+            # Create zvols
+            for c in crange('a', 'b'):
```

Comment:
Would it make sense to also create a zol that contains a space character (to exercise the `replace('+', ' ')` )?